src/test/kotlin/no/nav/familie/ef/sak/integration/PdlClientTest.kt

```diff
@@ -8,38 +8,47 @@ import io.mockk.mockk
 import no.nav.familie.ef.sak.config.PdlConfig
 import no.nav.familie.http.sts.StsRestClient
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.AfterEach
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.*
 import org.springframework.boot.web.client.RestTemplateBuilder
 import org.springframework.web.client.RestOperations
 import java.net.URI
 import java.time.LocalDate
 
 class PdlClientTest {
 
-    private val wireMockServer = WireMockServer(wireMockConfig().dynamicPort())
-    private val restOperations: RestOperations = RestTemplateBuilder().build()
-    private lateinit var pdlClient: PdlClient
+    companion object {
 
-    @BeforeEach
-    fun setUp() {
-        wireMockServer.start()
-        val stsRestClient = mockk<StsRestClient>()
-        every { stsRestClient.systemOIDCToken } returns "token"
-        pdlClient = PdlClient(PdlConfig(URI.create(wireMockServer.baseUrl())), restOperations, stsRestClient)
+        val restOperations: RestOperations = RestTemplateBuilder().build()
+        lateinit var pdlClient: PdlClient
```

Comment:
Disse burde vel kanskje være private?

Lurer på om det hadde vært tilstrekkelig å kun flytte .start() til en init{} og kun ha `resetAll`i en aferEach, for å ikke ha allt dette under companion object?

Reply:
`private` 👍

Init: Start Det må vel skje i en "beforeAll", eller kan du legge det på utsiden av alt? BeforeAll er static og følgelig må man knote litt i Kotlin. `start()` er void - så funker ikke å kjøre den initielt heller?

Tar gjerne imot kjørbart forslag hvis det går an å bli kvitt companion object her 🧐

Reply:
Aah tenkte feil, men ja det er riktig, må ha det i companion object! 👍